### PR TITLE
removed selected option from form builder

### PR DESCRIPTION
**Pull request by kristinh12**
Closes: https://github.com/lytics/pathforajs/issues/312

Removed the option to have things "selected" by default in the form builder since that's not supported in pathfora.  This was only present when you used "Add Option", not on the fields already displayed.

screen shot 2018-11-02 at 10 06 04 am

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "html-minifier": "^3.5.20",
     "jasmine-core": "^3.2.1",
     "jsdoc": "^3.5.5",
-    "karma": "^3.0.0",
+    "karma": "^3.1.1",
```
[the remark this answers is not in the document]
Comment (kristinh12) [reply package.json]: Hmm, probably accidental when I installed karma, but let me try it with the other version.

[the remark this answers is not in the document]
Comment (prosdev) [reply package.json]: We could prolly update this as a separate commit tho. If it doesn't break anything, always good to keep it fresh? @sarahyablok

[the remark this answers is not in the document]
Comment (sarahyablok) [reply package.json]: Yeah, I am pro-updating it, but yeah, seemed like it wasn't intentionally a part of this commit/PR. I am fine with it so long as tests pass, but agree lets make it in another commit.

[the remark this answers is not in the document]
Comment (kristinh12) [reply package.json]: Ok I just changed it back to 3.0.0, so now update it again in it's own commit?

[the remark this answers is not in the document]
Comment (kristinh12) [reply package.json]: or separate PR?

**Pull request merged**